packages/opentelemetry-plugin-xml-http-request/src/xhr.ts

```diff
@@ -14,51 +14,22 @@
  * limitations under the License.
  */
 
-import {
-  BasePlugin,
-  hrTime,
-  isUrlIgnored,
-  isWrapped,
-  otperformance,
-  urlMatches,
-} from '@opentelemetry/core';
+import { BasePlugin, hrTime, isWrapped, otperformance, isIgnored } from '@opentelemetry/core';
 import * as types from '@opentelemetry/types';
-import {
-  addSpanNetworkEvent,
-  getResource,
-  parseUrl,
-  PerformanceTimingNames as PTN,
-} from '@opentelemetry/web';
+import { PluginOptions, XMLHttpRequestPluginOptions, IgnoreMatcher } from '@opentelemetry/types';
+import { addSpanNetworkEvent, getResource, parseUrl, PerformanceTimingNames as PTN } from '@opentelemetry/web';
 import * as shimmer from 'shimmer';
 import { AttributeNames } from './enums/AttributeNames';
 import { EventNames } from './enums/EventNames';
 import { Format } from './enums/Format';
-import {
-  OpenFunction,
-  PropagateTraceHeaderCorsUrls,
-  SendFunction,
-  XhrMem,
-} from './types';
+import { OpenFunction, SendFunction, XhrMem } from './types';
 
 // how long to wait for observer to collect information about resources
 // this is needed as event "load" is called before observer
 // hard to say how long it should really wait, seems like 300ms is
 // safe enough
 const OBSERVER_WAIT_TIME_MS = 300;
 
-/**
```

Comment:
why did you remove that ?

Reply:
All plugin configurations are being centralized. That is the point of this PR